vochain/transaction/transaction.go

```diff
@@ -249,12 +249,17 @@ func (t *TransactionHandler) CheckTx(vtx *vochaintx.VochainTx, forCommit bool) (
 					); err != nil {
 						return nil, fmt.Errorf("setAccountTx: consumeFaucet %w", err)
 					}
+					if err := t.state.TransferBalance(&vochaintx.TransferTokensMeta{
+						FromAddress: faucetIssuerAddress,
+						ToAddress:   txSenderAddress,
+						Amount:      faucetPayload.Amount,
+						TxHash:      vtx.TxID[:],
+						Height:      uint64(t.state.CurrentHeight()),
```

Comment:
Why do we need the Height here and the transaction hash?

Reply:
For having this information available on the state that is later consumed by the indexer

Reply:
But the indexer already have access to the blockchain Height, I don't see the need for adding it here. Is there another reason behind this?